jujugui/static/gui/src/app/assets/css/base.scss

```diff
@@ -164,6 +165,14 @@ input[type="password"] {
     border-radius: $border-radius;
 }
 
+input[type=number]::-webkit-inner-spin-button,
+input[type=number]::-webkit-outer-spin-button {
```

Comment:
Does this mean we actually can style them? Or only in certain browsers? I could see them be there if we could get some styling that matched the UI